File: PYTHON/Pertemuan 01/PersegiLatihan.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
class PersegiLatihan:
    def __init__(self, sisi: float):
        self.sisi = sisi

    def hitung_luas(self) -> float:
        return self.sisi * self.sisi

    def hitung_keliling(self) -> float:
        return 4 * self.sisi

    def cetak(self):
        print(f"  Persegi (sisi={self.sisi:.1f}): Luas={self.hitung_luas():.2f}, Keliling={self.hitung_keliling():.2f}")

File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/PersegiLatihan.java
package id.ac.uph.pbo.pertemuan01;

public class PersegiLatihan {
    private double sisi;
    public PersegiLatihan(double sisi) { this.sisi = sisi; }
    public double hitungLuas()     { return sisi * sisi; }
    public double hitungKeliling() { return 4 * sisi; }
    public void cetak() {
        System.out.printf("  Persegi (sisi=%.1f): Luas=%.2f, Keliling=%.2f%n",
                sisi, hitungLuas(), hitungKeliling());
    }
}
